content/articles/edns-client-subnet.markdown

```diff
@@ -0,0 +1,45 @@
+---
+title: EDNS Client Subnet support for ALIAS records
+excerpt: What ECS is, and how it works.
+categories:
+- DNS
+---
+
+# EDNS Client Subnet support for ALIAS records
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+This article explains what EDNS Client Subnet support is, and how it's configured with an ALIAS record.
```

Comment:
EDNS is never defined. Perhaps it makes sense to define what EDNS is in another support article and link to that?

Reply:
@aeden: I will put another technical support article for EDNS, but will take some time to go over the creation/approval process. I suggest we go ahead and publish as it is and can amend this one once the EDNS support page is up?